retrace/metric_backend_opengl.cpp

```diff
@@ -83,6 +85,8 @@ MetricBackend_opengl::MetricBackend_opengl(glretrace::Context* context,
     supportsElapsed     = context->hasExtension("GL_EXT_timer_query") || supportsTimestamp;
     supportsOcclusion   = currentProfile.versionGreaterOrEqual(glfeatures::API_GL, 1, 5);
 
+    glGetQueryiv(GL_TIMESTAMP, GL_QUERY_COUNTER_BITS, &timestamp_bits_precision);
```

Comment:
We should add here

```c++
    if (timestamp_bits_precision >= 64) {
         timestamp_bits_precision = 0;
    }
```
Because we our counters are also 64 bits.